libs/platform/src/lib/components/menu-button/menu-button.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MenuButtonComponent } from './menu-button.component';
+import { ButtonModule } from '@fundamental-ngx/core';
+
+describe('MenuButtonComponent', () => {
+    let component: MenuButtonComponent;
+    let fixture: ComponentFixture<MenuButtonComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            imports: [ButtonModule],
+            declarations: [MenuButtonComponent],
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(MenuButtonComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should have a disabled menu button', () => {
+        spyOn(component, 'setDisabledState');
+        component.setDisabledState(true);
+        expect(component.setDisabledState).toHaveBeenCalled();
+    });
+
```

Comment:
I would add tests for compact mode and for type (check if type is applied)

Reply:
will do.